packages/flutter_tools/lib/src/asset.dart

```diff
@@ -248,14 +250,30 @@ class _ManifestAssetBundle implements AssetBundle {
       _wildcardDirectories[uri] ??= globals.fs.directory(uri);
     }
 
+    final List<File> newAdditionalDependencies = <File>[];
+
+    // Include the SkSl shader files in shaders/
+    if (includeShaderFiles && globals.fs.directory('shaders').existsSync()) {
```

Comment:
What if some Flutter project has a user-created `shaders` directory in the root Flutter directory? Would it be safer to put such directory under `build` so it's clear that the files are auto-generated and not user-created?

Reply:
Yeah thats a good idea - these should probably be versioned and in the build artifacts directory